util/options_helper.h

```diff
@@ -625,6 +625,9 @@ static std::unordered_map<std::string, OptionTypeInfo>
         {"index_per_partition",
          {offsetof(struct BlockBasedTableOptions, index_per_partition),
           OptionType::kUInt64T, OptionVerificationType::kNormal, false, 0}},
+        {"partition_filters",
```

Comment:
alright. Let's comment in table.h that partition_filters works only with kTwolevelIndexSearch right now.